packages/hardhat/contracts/GardensTemplate.sol

```diff
@@ -258,8 +272,8 @@ contract GardensTemplate is BaseTemplate, AppIdsXDai {
         acl.createPermission(collateralRequirementUpdater, agreement, agreement.MANAGE_DISPUTABLE_ROLE(), disputableVoting);
```

Comment:
What you think of including a note here of why we need this permission here?

Reply:
Added